Tests/Tests/STPPaymentHandlerStubbedMockedFilesTests.swift

```diff
@@ -184,12 +188,13 @@ class STPPaymentHandlerStubbedMockedFilesTests: APIStubbedTestCase, STPAuthentic
         paymentIntentParams.paymentMethodParams?.afterpayClearpay =
             STPPaymentMethodAfterpayClearpayParams()
         let didRedirect = expectation(description: "didRedirect")
-        paymentHandler._redirectShim = { redirectTo, returnToURL in
+        paymentHandler._redirectShim = { redirectTo, returnToURL, callingContext in
             XCTAssertEqual(
                 redirectTo.absoluteString,
                 "https://hooks.stripe.com/afterpay_clearpay/acct_123/pa_nonce_321/redirect"
             )
             XCTAssertEqual(returnToURL?.absoluteString, "payments-example://stripe-redirect")
+            XCTAssert(callingContext.starts(with: "_handleRedirect("))
```

Comment:
This is kinda lurky, is there a way to type this more strongly?

Reply:
I was thinking to introduce an enum to get some more type safety, but I didn't want to increase our binary size by introducing a new type.  Another option I thought of was introducing a new shim, but that seemed worse -- hmm, I guess since there are only two cases, I could introduce a boolean -- one that signifies that it is a regular redirect vs an external browser redirect?

Reply:
If we need more cases, we could eventually turn it into an integer, but for now, a boolean seems appropriate?

Reply:
Bool is fine! And we don't need to be *that* careful with enums w/r/t binary size, especially if they don't conform to Codable or Equatable or anything that would cause the compiler to generate more boilerplate.

Reply:
Are we using this value anywhere other than tests?
I agree a boolean is better than using `#function`, if we were using it outside of tests I would advocate for the enum.

Reply:
No, this is value is only being used for tests.  great. i'll switch over to using a bool :)